src/smc-webapp/editor_chat.cjsx

```diff
@@ -785,7 +790,9 @@ ChatRoom = (name) -> rclass
     on_scroll: (e) ->
```

Comment:
So, my thought is, that e.g. right here on GitHub you can open up the development console and type in `document.onscroll = function() { console.log("x"); }`.
Once you scroll even a little bit, you see that function is fired a hundred times. 

[throttle/debounce](http://underscorejs.org/#throttle) would either rate-limit this or defer it until after scrolling is over. I'm thinking of 50ms or so ...

I should check though, if that even applies here. It's just that his is one of those things where scrolling is suddenly getting laggy, etc.